lib/Ohmbrewer_Onewire.cpp

```diff
@@ -124,7 +142,7 @@ void Ohmbrewer::Onewire::displayProbeIds(Ohmbrewer::Screen *screen){
     //If your code has other tasks, you can store the timestamp instead and return when a second has passed.
     delay(1000);
     uint8_t numSensors = ow_search_sensors(10, sensors);
-    screen->displayHeader();
+
     for (uint8_t i=0; i<numSensors; i++){
         //current probe ID
         char probeId[8] = {
```

Comment:
I think we technically would want to use ```byte``` instead of ```char``` here, since we're working with unsigned HEX values.